Krivorotova Tatyana/flex_layout/sass.css

```diff
@@ -0,0 +1,123 @@
+body {
```

Comment:
if you compile your `css` files from `scss` - there is no need to commit `css` - just provide scripts for compiling in `package.json`